### ISSUE TITLE
'Potentially Dangerous alternative prefix'

### ISSUE THREAD

**Issue by LemADEC** [Da-Technomancer/Essentials#5]
As of essentials-1.12.2-1.2.0, forge is reporting warnings in the console:
```
[00:34:24] [main/WARN] [FML]: Potentially Dangerous alternative prefix `minecraft` for name `crossroads_brazier`, expected `essentials`. This could be a intended override, but in most cases indicates a broken mod.
[00:34:24] [main/WARN] [FML]: Potentially Dangerous alternative prefix `minecraft` for name `crossroads_slottedchest`, expected `essentials`. This could be a intended override, but in most cases indicates a broken mod.
[00:34:24] [main/WARN] [FML]: Potentially Dangerous alternative prefix `minecraft` for name `crossroads_sortinghopper`, expected `essentials`. This could be a intended override, but in most cases indicates a broken mod.
[00:34:24] [main/WARN] [FML]: Potentially Dangerous alternative prefix `minecraft` for name `crossroads_itemchuteport`, expected `essentials`. This could be a intended override, but in most cases indicates a broken mod.
[00:34:24] [main/WARN] [FML]: Potentially Dangerous alternative prefix `crossroads` for name `port_extender`, expected `essentials`. This could be a intended override, but in most cases indicates a broken mod.
```

Some of those seem to overlap with your crossroads mod, so you probably need to fix both in one shot :).

**Comment by Da-Technomancer** [comment]
This is a known "issue". The tile entity IDs were not changed when moving features from Crossroads, as changing them would corrupt existing worlds. As 1.13 will be a world-breaking update most likely anyway, the IDs will be changed to the correct mod ids. 

In the meantime, having the wrong ID doesn't really break anything other than spamming startup logs.

**Comment by Da-Technomancer** [comment]
Fixed in the 1.13 version.
